packages/modals/src/scrollable_modal/index.stories.tsx

```diff
@@ -27,7 +27,7 @@ export const Default = () => {
           staticPosition={staticPosition}
         >
           <ul>
-            {[...(new Uint8Array(numberOfItems))].map((value, key) => (
+            {[...(new Uint8Array(numberOfItems))].map((_, key) => (
```

Comment:
do we define that _ as argsIgnorePattern or is it from airbnb?

Reply:
It's actually from airbnb. I agree with you, it's a non-descriptive.  I should just rename from `_` to `_value`, that's prolly clearer.